README.md

```diff
@@ -24,7 +24,10 @@ Here are some of the benefits of using this library:
 Feature flags are composed of two parts, a name and a list of feature-filters that are used to turn the feature on.
 
 ### Feature Filters
-Feature filters define a scenario for when a feature should be enabled. When a feature is evaluated for whether it is on or off, its list of feature-filters are traversed until one of the filters decides the feature should be enabled. At this point the feature is considered enabled and traversal through the feature filters stops. If no feature filter indicates that the feature should be enabled, then it will be considered disabled.
+Feature filters define a scenario for when a feature should be enabled. When a feature is evaluated for whether it is on or off, its list of feature-filters are traversed until one of the filters decides the feature should be enabled or disabled. Traversal works in one of two ways:
+
+1. For filters with a `RequirementType` of `Any` (the default): The feature-filters are traversed until one of the filters returns true. If no filter returns true then the feature is disabled.

```

Comment:
> until one of the filters returns true

Instead of saying that it "returns true", can we say "is enabled"?
Returning a particular value is implementation detail.

So the sentence would be something like:
"Feature filters are traversed until one of the filters decides that the feature should be enabled. If no filter indicates that the feature should be enabled, then it will be considered disabled."

For RequirementType All:
"Feature filters are traversed until one of the filters decides that the feature should be disabled. If no filter indicates that the feature should be disabled, then it will be considered enabled."

I dont think we need to mention that there should be at least one filter to enable the feature. That information is already communicated elsewhere in the doc and its not specific to Requirement Type.